tests/next_tests/integration_tests/feature_tests/ffront_tests/ffront_test_utils.py

```diff
@@ -229,6 +229,7 @@ def simple_mesh() -> MeshDescriptor:
             C2E.value: gtx.NeighborTableOffsetProvider(
                 c2e_arr, Cell, Edge, 4, has_skip_values=False
             ),
+            Koff.value: KDim,
```

Comment:
Why is this change here?

Reply:
This is an easy way to trigger the error.

Reply:
But not the best I guess. An isolated test would be one that has an Cartesian offset for a dimension that is not in any field?

Reply:
any updates on this? I would need it for https://github.com/C2SM/icon4py/pull/564

Reply:
@nfarabullini Let's introduce a new `unstructured_case_3d` fixture that starting from the `unstructured_case` fixture adds the default size and offset provider for the `KDim`.
@jcanton I think we can merge this soon, but this will likely not help you much since icon4py is pinned to an older version of GT4PY until https://github.com/GridTools/gt4py/pull/1738 is merged.

Reply:
I didn't think about that, thanks!